.drone.yml

```diff
@@ -964,3 +954,59 @@ steps:
         from_secret: discord_webhook_id
       webhook_token:
         from_secret: discord_webhook_token
+
+---
+kind: pipeline
+name: test-env-linux-amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+trigger:
+  event:
+  - promote
```

Comment:
a manually triger. I think it's better use it in here
https://docs.drone.io/promote/